e2e_tests/test_suites/guestpolicies/guest_policies_utils.go

```diff
@@ -243,21 +258,21 @@ baseurl=https://packages.cloud.google.com/yum/repos/osconfig-agent-test-reposito
 enabled=1
 gpgcheck=0
 EOM
-zypper -n remove %[2]s
-zypper -n --no-gpg-checks install %[2]s-3.03-2.fc7
+zypper -n remove ed
```

Comment:
Is there a chance we could get a false signal if inventory including `ed` is already reported before we remove it?

Reply:
It's not actually installed on anything but centos6, i put these here in case a future image has it. Also we don't look for it until after we remove it and restart the agent